all/099-generic.yml

```diff
@@ -165,6 +165,12 @@ proxy_proxies: {}
 cleanup_services_default:
   - ModemManager.service
 
+cleanup_packages_default:
```

Comment:
nit: p<s

also maybe add the comment from the commit message here for future reference?